one-ui/src/main/java/com/marklogic/hub/oneui/auth/MarkLogicAuthenticationManager.java

```diff
@@ -113,8 +114,12 @@ public Authentication authenticate(Authentication authentication) throws Authent
             stagingServerAccessible = dataServicesClient.checkConnection().isConnected();
         } catch (Exception e) {
         }
+
+        String projDir = environmentService.getProjectDirectory();
+        projDir = projDir.substring(projDir.lastIndexOf(File.separator) + 1);
```

Comment:
This is really "project name", right? Not the project directory?

Reply:
Yeah, but we don't have a "project name" currently. I talked to @ryanjdew  and he suggested that we use the project directory for now.